seed_data/management/commands/create_tiers_test.py

```diff
@@ -59,3 +59,11 @@ def test_create_tier_with_add_to_existing_parameter(self):
         self.command.handle("create_tiers", tiers=4, program_id=None, add_to_existing=True)
         assert TierProgram.objects.filter(program=self.program1).count() == 4  # Default is 4
         assert TierProgram.objects.filter(program=self.program2).count() == 4
+
+    def test_zero_threshold_and_discount(self):
+        """
+        Test that there is one TierProgram with income_threshold=0 and one with discount_amount=0
+        """
+        self.command.handle("create_tiers", tiers=4, program_id=None, add_to_existing=False)
```

Comment:
I don't think we need to make sure that the two entries are different, necessarily. `income_threshold` needs to be at `0` because it sets a minimum income threshold for financial aid and prevents undefined behavior where the users income is below the minimum tier. `discount_amount` needs to be `0` because if a user skips financial aid, they are still set to some financial aid tier, so this one gives `0` discount.

It is absurd for the `$0` income threshold to be the same as the `$0` discount since it means the user with the least income pays the highest price, but for testing purposes it should be fine.